www/app/offer/offer.controller.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('smartquote.offer')
+    .controller('OfferController', OfferController);
+
+  OfferController.$inject = [];
+
+  /* @ngInject */
+  function OfferController() {
```

Comment:
If you do that, the dependencies of the OfferController are not injected. Please check the categories controller